azurerm/internal/services/firewall/firewall_policy_resource.go

```diff
@@ -211,6 +224,12 @@ func resourceFirewallPolicyCreateUpdate(d *pluginsdk.ResourceData, meta interfac
 		}
 	}
 
+	if v, ok := d.GetOk("private_ip_ranges"); ok {
+		privateIpRanges := utils.ExpandStringSlice(v.([]interface{}))
+		props.FirewallPolicyPropertiesFormat.Snat = &network.FirewallPolicySNAT{
+			PrivateRanges: privateIpRanges}
```

Comment:
_minor_ can we clean this up:

```suggestion
			PrivateRanges: privateIpRanges,
		}
```